delta/utils/decode/tf_ctc.py

```diff
@@ -46,8 +46,27 @@ def ctc_decode_data_prepare(logits, sequence_length, blank_id=None):
       false_fn=lambda: tf.squeeze(sequence_length),
   )
 
-  return logits_return, sequence_length_return
+  return logits_return, sequence_length_return, blank_id
 
+def ctc_decode_post_process(decode_result, blank_id):
+  '''
+    change the value of blank_label elements from num_classes - 1 to blank_id
+    param: decode_result, an tf.SparseTensor, containing the docode result
+    param: blank_id, int, the index of blank label
+    return: decode_result_return: an tf.SparseTensor
+    '''
+  if blank_id < 0 or blank_id is None:
+    raise ValueError('blank_index must be greater than or equal to zero')
+
+  decode_labels = decode_result.values
+  labels_change_blank_id = tf.where(decode_labels >= blank_id,
```

Comment:
Does not consider `num_classes-1` to `blank_id`

Reply:
The value of num_classes-1 cannot be determined according to the decode_result